src/articles/0006-choropleth-on-d3v4/choropleth.js

```diff
@@ -62,39 +36,12 @@ class choroplethBase extends React.Component {
     d3.select("svg").remove();
   }
 
-  renderText() {
-    return (
-      <div
-        className="content has-text-grey-darker"
-        dangerouslySetInnerHTML={{
-          __html: this.props.data.markdownRemark.html
-        }}
-      />
-    );
-  }
-
   render() {
     return (
-      <BlogPost
-        componentBlocks={[
-          {
-            wrapper: "break-out",
-            uniqueKey: "states",
-            renderComponent: () => <div id="states" />
-          },
-          {
-            wrapper: "break-out",
-            uniqueKey: "tooltip",
-            renderComponent: () => <div id="tooltip" />
-          },
-          {
-            wrapper: "column",
-            uniqueKey: "text-column",
-            renderComponent: this.renderText.bind(this)
-          }
-        ]}
-        {...this.props}
-      />
+      <React.Fragment>
```

Comment:
You could simplify this a bit with
```
<>
  <div id="states" />
  <div id="tooltip" />
</>
```

Reply:
I prefer to be explicit with that as it's really hard to search on find what `<>` does if you don't already know.